modules/core/misc/java/src/java/core+MatOfByte.java

```diff
@@ -30,24 +30,37 @@ public MatOfByte(Mat m) {
         //FIXME: do we need release() here?
     }
 
-    public MatOfByte(byte...a) {
+	public MatOfByte(byte...a) {
```

Comment:
Please restore indentation (tabs -> 4 spaces) here and below.

Reply:
Oops sorry.